### PR TITLE
fix(xamlreader): Support for geometry parsing

### PR DESCRIPTION
**Pull request by jeromelaban**
GitHub Issue (If applicable): closes https://github.com/unoplatform/uno/issues/10577

## PR Type

What kind of change does this PR introduce?
- Feature

## What is the new behavior?

Support for `XamlReader` `Geometry` parsing.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/Given_XamlReader.cs [hunk 1]
```diff
@@ -1458,6 +1459,37 @@ public void When_StaticResource_MergedDictionaries_Fluent()
...
+				leftCorner,
+				leftCorner,
+				actualHeight - (leftCorner + rightCorner + 1.0f / scaleFactor),
+				rightCorner,
+				rightCorner,
+				rightCorner,
+				rightCorner,
+				height - (5 + rightCorner));
+
+			var geometry = Windows.UI.Xaml.Markup.XamlReader.Load(strOut) as Uno.Media.StreamGeometry;
```
Comment (Youssef1313) [inline src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/Given_XamlReader.cs:1488]: This doesn't quite match the behavior on Windows.

On Windows the returned geometry is PathGeometry

Comment (jeromelaban) [reply src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/Given_XamlReader.cs:1488]: Correct, but we're not going to fix this one for this PR.

File: src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/Given_XamlReader.cs [hunk 2]
```diff
@@ -1458,6 +1459,37 @@ public void When_StaticResource_MergedDictionaries_Fluent()
 			Assert.AreEqual(t1, b1.Color);
 		}
 
+		[TestMethod]
+		public void MyTestMethod()
```
Comment (Youssef1313) [inline src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/Given_XamlReader.cs:1463]: I assume the test name isn't intended

File: src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/Given_XamlReader.cs [hunk 3]
```diff
@@ -1458,6 +1459,37 @@ public void When_StaticResource_MergedDictionaries_Fluent()
...
+				leftCorner,
+				leftCorner,
+				leftCorner,
+				leftCorner,
+				actualHeight - (leftCorner + rightCorner + 1.0f / scaleFactor),
+				rightCorner,
+				rightCorner,
+				rightCorner,
+				rightCorner,
+				height - (5 + rightCorner));
```
Comment (Youssef1313) [inline src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/Given_XamlReader.cs:1486]: For the purpose of the test, I don't think we need all these variables or calling `string.Format`, it's easier to understand the test by having the exact string we want to pass to XamlReader

Comment (MartinZikmund) [reply src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/Given_XamlReader.cs:1486]: Yes, sorry, I took the test out of the TabView code

**Review by MartinZikmund — APPROVED** [review]

File: src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/Given_XamlReader.cs (hunk 3 above)
Comment (jeromelaban) [reply src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/Given_XamlReader.cs:1486]: Yes it can all be removed.

**Pull request merged**